pkg/executor/infoschema_reader.go

```diff
@@ -3801,9 +3741,8 @@ func (e *memtableRetriever) setDataFromIndexUsage(ctx context.Context, sctx sess
 	return nil
 }
 
-// Currently, ClusterIndexUsage will not be be pushed down, so just use the previous logic
-func (e *memtableRetriever) setDataFromClusterIndexUsage(ctx context.Context, sctx sessionctx.Context, schemas []model.CIStr) error {
-	err := e.setDataForClusterIndexUsage(ctx, sctx, schemas)
+func (e *memtableRetriever) setDataFromClusterIndexUsage(ctx context.Context, sctx sessionctx.Context) error {
+	err := e.setDataFromIndexUsage(ctx, sctx)
```

Comment:
What's the difference between clustered_index_usage table and tidb_index_usage table?

Reply:
OK, different tidb at the same time may see different schema.

Reply:
https://docs.pingcap.com/tidb/stable/information-schema-tidb-index-usage#tidb_index_usage

I think each tidb will collect its own index-related statistics.

Reply:
For example, I ran some query on one node and got this output.
```
mysql> select * from information_schema.CLUSTER_TIDB_INDEX_USAGE where table_schema = "test" and index_name = "idx_1";
+-----------------+--------------+------------+------------+-------------+--------------+-------------------+---------------------+-----------------------+------------------------+-------------------------+-------------------------+--------------------------+-----------------------+---------------------+
| INSTANCE        | TABLE_SCHEMA | TABLE_NAME | INDEX_NAME | QUERY_TOTAL | KV_REQ_TOTAL | ROWS_ACCESS_TOTAL | PERCENTAGE_ACCESS_0 | PERCENTAGE_ACCESS_0_1 | PERCENTAGE_ACCESS_1_10 | PERCENTAGE_ACCESS_10_20 | PERCENTAGE_ACCESS_20_50 | PERCENTAGE_ACCESS_50_100 | PERCENTAGE_ACCESS_100 | LAST_ACCESS_TIME    |
+-----------------+--------------+------------+------------+-------------+--------------+-------------------+---------------------+-----------------------+------------------------+-------------------------+-------------------------+--------------------------+-----------------------+---------------------+
| 127.0.0.1:10081 | test         | t          | idx_1      |           0 |            0 |                 0 |                   0 |                     0 |                      0 |                       0 |                       0 |                        0 |                     0 | NULL                |
| 127.0.0.1:10080 | test         | t          | idx_1      |           1 |            1 |                 3 |                   0 |                     0 |                      0 |                       0 |                       0 |                        1 |                     0 | 2024-08-15 09:46:04 |
+-----------------+--------------+------------+------------+-------------+--------------+-------------------+---------------------+-----------------------+------------------------+-------------------------+-------------------------+--------------------------+-----------------------+---------------------+
```